core/domain/user_jobs_one_off_test.py

```diff
@@ -1354,6 +1354,72 @@ def test_that_last_edited_and_created_time_are_not_updated(self):
         self.assertIsNone(owner_settings.last_edited_an_exploration)
 
 
+class FillExplorationIdsInUserSubscriptionsModelOneOffJobTests(
```

Comment:
I suggest also testing the scenario where a model already exists with exploration IDs already populated/running the job twice to ensure it doesn't perform redundant actions.

Reply:
Done.

Reply:
To be slightly more clear, I think the new tests look good but we should also have a test specifically covering the run-job-twice scenario since it's important to make sure that this is an option (migration jobs shouldn't leave the data store in a bad state if a failure is encountered & they need to be re-run).